### PR TITLE
solution first draft

### PR DESCRIPTION
**Pull request by scottcreynolds**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,50 @@
...
+  const xhr = new XMLHttpRequest()
+  xhr.addEventListener("load", displayRepositories)
+  xhr.open("GET", uri)
+  xhr.send()
+  return false;
+}
+function displayRepositories() {
+  const repos = JSON.parse(this.responseText)
+  const repoList = "<ul>" + repos.map(repo => {
+    let dataRepoName = 'data-repository="' + repo.name + '"'
```
Comment (pletcher) [inline index.js]: Could we use `const` here?

File: index.js [hunk 2]
```diff
@@ -1 +1,50 @@
...
+    let dataUsername = 'data-username="' + repo.owner.login + '"'
+    let str = '<li><h2>' + repo.name + '</h2>'
```
Comment (pletcher) [inline index.js]: Could we make this a template literal? Multiline strings are allowed, e.g.,

``` js
return (`
  <li>
    <h2>${repo.name}</h2>
    <a href="${repo.html_url}">${repo.html_url}</a><br/>
    // etc.
  </li>
`)
```

**Comment by pletcher** [comment]
This is great! I left some minor notes on one function in the solution, but otherwise LGTM!

**Comment by scottcreynolds** [comment]
@pletcher updates made

**Comment by pletcher** [comment]
LGTM!

**Pull request merged**